### ISSUE TITLE
Add watch update notifications

### ISSUE THREAD

**Issue by skylarmt** [AsteroidOS/AsteroidOSSync#37]
It would be nice if I could get a notification on my phone when new Asteroid builds with significant changes (bug fixes, new features) are available for download.  That way I won't have to go through the IRC logs or GitHub commit history every few days.

**Comment by Huntereb** [comment]
I'm sure that's planned for the future when new Non-Alpha builds would be available, but as of right now I don't think it's something effort should be pushed toward. It would be a lot of extra work for something that can currently be obtained from IRC logs or modifications pushed to github, as you mentioned.

To make things easier, you can figure out if a nightly has been recently built by looking at the dates of the files available for your device [where the nightlies are hosted](https://release.asteroidos.org/nightlies/dory/).

**Comment by KorakSilvercloud** [comment]
I agree that this is a good feature for the future. Would be hard to do currently due to the only builds being nightly Alpha builds. Ideally this would be put in place to notify you each time there is a new stable build. Currently though the project is working towards the first official stable build.

**Comment by jrtberlin** [comment]
For this, we probably need a [BLE profile](https://asteroidos.org/wiki/ble-profiles/) in [asteroid-btsyncd](https://github.com/AsteroidOS/asteroid-btsyncd) that returns the version that is currently installed. We already have the [watch name ](https://github.com/AsteroidOS/AsteroidOSSync/blob/master/app/src/main/java/org/asteroidos/sync/fragments/DeviceListFragment.java#L170) in the app. But I don't know if that is always the codename so I would opt to return the codename in its own Characteristic.
